ledger/accountdb.go

```diff
@@ -4809,6 +4816,12 @@ func (prd *persistedResourcesData) before(other *persistedResourcesData) bool {
 	return prd.round < other.round
 }
 
+// before compares the round numbers of two persistedKVData and determines if the current persistedKVData
+// happened before the other.
+func (prd *persistedKVData) before(other *persistedKVData) bool {
```

Comment:
Since they are small objects, I might not use pointers for the receiver or the arg.